client/my-sites/plan-features-2023-grid/hooks/use-highlight-label.ts

```diff
@@ -5,16 +5,21 @@ import {
 } from '@automattic/calypso-products';
 import { useTranslate } from 'i18n-calypso';
 import { useSelector } from 'react-redux';
+import { NEWSLETTER_FLOW } from 'calypso/../packages/onboarding/src';
 import { getCurrentPlan } from 'calypso/state/sites/plans/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
-const useHighlightLabel = ( planName: string ) => {
+const useHighlightLabel = ( planName: string, flowName?: string ) => {
 	const translate = useTranslate();
 	const selectedSiteId = useSelector( getSelectedSiteId );
 	const currentPlan = useSelector( ( state ) => getCurrentPlan( state, selectedSiteId ) );
 	const isCurrentPlan = currentPlan?.productSlug === planName;
 
-	if ( isCurrentPlan ) {
+	if ( flowName === NEWSLETTER_FLOW ) {
```

Comment:
I know the processing here may change (per James' review), but for such conditions, I think we have respective selectors right? e.g. `isNewsletterFlow`